docs/api_clients/base.rst

```diff
@@ -0,0 +1,17 @@
+Base Client
+===========
+
+.. currentmodule:: enlyze.api_clients.base
+
+.. autoclass:: M
+
+.. autoclass:: R
+
+.. autoclass:: ApiBaseModel
+
+.. autoclass:: PaginatedResponseBaseModel
+
+.. autoclass:: ApiBaseClient
```

Comment:
Can we hide `_abc_impl` from the docs?
![image](https://github.com/enlyze/enlyze-python/assets/224128/671b55c3-e1fb-46be-8c44-0acb763115bd)